dipper/sources/ZFIN.py

```diff
@@ -2495,19 +2439,19 @@ def process_fish_disease_models(self, limit=None):
         return
 
     def _map_sextuple_to_phenotype(
-            self, superterm1_id, subterm1_id, quality_id, superterm2_id,
-            subterm2_id, modifier):
+            self,
+            superterm1_id, subterm1_id, quality_id,
```

Comment:
This will probably work, but maybe use the relationship columns as well.

Reply:
I actually did use the relationship columns in a [previous commit on this branch](https://github.com/monarch-initiative/dipper/pull/735/commits/3b621c278caf8312bece884651cc9c3caaf565d9#diff-57a8cd3fe591cdb53eebc04f55bc8295L2497)
Could roll back and do that, but my concern is that if we add these two relationships columns we will lose some g2p triples that are present in the current ingest if even a single one of the relationship ids doesn't match up to what's in phenotype_fish.txt or phenoGeneCleanData_fish.txt

Reply:
Yes, but this is exactly the reason they exist: they should not match up. Basically, if it is not in the map, it should not be ingested; so rather than allowing here to map to incomplete records (without the relationships), we need to continue to work expanding the coverage of these mappings in the ZP further..

Reply:
Ah okay - duly noted, thanks Nico